src/lib/testrail.ts

```diff
@@ -44,23 +45,23 @@ export class TestRail {
     }
     return this.makeSync(
       axios({
-        method:'get',
+        method: 'get',
         url: url,
-        headers: { 'Content-Type': 'application/json' }, 
+        headers: { 'Content-Type': 'application/json' },
         auth: {
-            username: this.options.username,
-            password: this.options.password
-        } 
-      })
-      .then(response => {
-        return response.data.map(item =>item.id)
+          username: this.options.username,
+          password: this.options.password
+        }
       })
-      .catch(error => console.error(error))
+        .then(response => {
+          return response.data.map(item => item.id)
+        })
```

Comment:
To keep it a bit more concise:

```suggestion
        .then(response => response.data.map(item => item.id))
```